modules/moderation.py

```diff
@@ -45,17 +62,36 @@ def validate_token(token: str) -> bool:
         user_id = int(base64.b64decode(user_id + "==", validate=True))
     except (ValueError, binascii.Error):
         return False
-    else:
-        return True
+    return True
 
 
 class GithubError(commands.CommandError):
     pass
 
 
+class ModerationRespostView(discord.ui.View):
```

Comment:
Should we disable the buttons once this times out, to make it clear that it has done so?